docs/jwst/references_general/abvega_offset_reffile.inc

```diff
@@ -0,0 +1,67 @@
+ABVEGA_OFFSET Reference File
+----------------------------
+
+:REFTYPE: ABVEGA_OFFSET
+
+The ABVEGA_OFFSET reference file contains data necessary for converting
+from AB to Vega magnitudes.
+
+.. include:: ../references_general/abvega_offset_selection.inc
+
+.. include:: ../includes/standard_keywords.inc
+
+APVEGA_OFFSET Reference File Format
++++++++++++++++++++++++++++++++++++
+
+ABVEGA_OFFSET reference files are in ASDF format.  The ABVEGA_OFFSET
+reference file contains tabular data in a key called
+``abvega_offset``.  The content of the table varies for different
+instrument modes, as shown in the tables below.
+
+:Data model: `~jwst.datamodels.ABVegaOffsetModel`
+
++------------+-------+---------------+-----------+------------+----------+
+| Instrument | Mode  | Column name   | Data type | Dimensions | Units    |
++============+=======+===============+===========+============+==========+
+| FGS        | Image | detector      | string    | 7          | N/A      |
++            +       +---------------+-----------+------------+----------+
+|            |       | abvega_offset | float     | scalar     | unitless |
++------------+-------+---------------+-----------+------------+----------+
+
++------------+-------+---------------+-----------+------------+----------+
+| Instrument | Mode  | Column name   | Data type | Dimensions | Units    |
++============+=======+===============+===========+============+==========+
+| MIRI       | Image | filter        | string    | 12         | N/A      |
++            +       +---------------+-----------+------------+----------+
+|            |       | abvega_offset | float     | scalar     | unitless |
++------------+-------+---------------+-----------+------------+----------+
+
++------------+-------+---------------+-----------+------------+----------+
+| Instrument | Mode  | Column name   | Data type | Dimensions | Units    |
++============+=======+===============+===========+============+==========+
+| NIRCam     | Image | filter        | string    | 12         | N/A      |
++ or         +       +---------------+-----------+------------+----------+
+| NIRISS     |       | pupil         | string    | 15         | N/A      |
++            +       +---------------+-----------+------------+----------+
+|            |       | abvega_offset | float     | scalar     | unitless |
++------------+-------+---------------+-----------+------------+----------+
+
+
+Row Selection
+^^^^^^^^^^^^^
+
+A row of data within the reference table is selected by the pipeline
+step based on the optical elements in use for the exposure. The
+selection attributes are always contained in the first few columns of
+the table.  The last column contains the data needed to convert from
+AB to Vega magnitudes.  The row selection criteria for each
+instrument/mode are:
+
+* FGS Image:
```

Comment:
Minor nit, but probably don't need all the explicit references to "Image" mode in each of these entries, because this ref file only applies to imaging mode to begin with (as opposed to the apcorr or photom ref files).

Reply:
Done.